src/components/presentational/ResourceList/ResourceList.tsx

```diff
@@ -35,7 +35,6 @@ export default function (props: ResourceListProps) {
                     imageUrl={resource.imageUrl}
                     onClick={() => props.onViewResource(resource)}
                     imageAlignment={props.imageAlignment}
-                    hideButton={!props.buttonVariant}
```

Comment:
@greinard 👆 ?

Reply:
Yeah, I must have missed removing this line at some point.  The absence of a button variant means to hide the button now, so it was probably just leftover during that refactor.

Removing this is correct.  👍